client/my-sites/patterns/components/category-gallery/client.tsx

```diff
@@ -68,12 +68,15 @@ export const CategoryGalleryClient: CategoryGalleryFC = ( {
 								>
 									<div className="patterns-category-gallery__item-preview-inner">
 										<PatternPreview
+											category={ category.name }
 											className="pattern-preview--category-gallery"
 											pattern={
 												patternTypeFilter === PatternTypeFilter.PAGES
 													? category.pagePreviewPattern
 													: category.regularPreviewPattern
 											}
+											patternTypeFilter={ patternTypeFilter }
+											view="grid"
```

Comment:
I see... Let's add a comment after it, since it can confuse devs who will works with it in the future, or even us when we forget about it in some time :)